hack/test-mulinode.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Ensure that subshells inherit bash settings (specifically xtrace)
+export SHELLOPTS
+
+OS_ROOT="${BASH_SOURCE%/*}/.."
+OS_ROOT="$(readlink -ev "${OS_ROOT}")"
+
+source "${OS_ROOT}/hack/text.sh"
+source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/hack/common.sh"
+source "${OS_ROOT}/hack/lib/log.sh"
+source "${OS_ROOT}/hack/lib/util/environment.sh"
+os::log::install_errexit
+
+CLUSTER_CMD="${OS_ROOT}/hack/dind-cluster.sh"
+CONFIG_ROOT="${OPENSHIFT_CONFIG_ROOT:-${OS_ROOT}}"
+
+# Use a unique instance prefix to ensure the names of the test dind
+# containers will not clash with the names of non-test containers.
+export OPENSHIFT_INSTANCE_PREFIX="testing"
+export OPENSHIFT_NUM_MINIONS=3
```

Comment:
I'll make it configurable, but that's exactly MINIONS because dind have one more node on master.